apps/data/database/Dockerfile

```diff
@@ -2,6 +2,8 @@ FROM postgres:16-alpine
 
 COPY ./postgresql.conf /etc/postgresql.conf
 
+COPY recommender/trainingData.csv /docker-entrypoint-initdb.d/recommender/
```

Comment:
Are these items being added to the legitimate database or only for the locally seeded dev instance? This seems like populating the local dev env but not adding to production, so this should probably be added to the docker compose path share and not the Dockerfile

Reply:
I believe we will just be needing this in test, so it is fine to just leave the volume I added for the csv in the docker-compose (as that yml won't be used for production) and remove this redundant copy?

Reply:
Yep